cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/MultiOpsPostgresSinkConsumerIT.java

```diff
@@ -97,7 +97,9 @@ public void verifyInsertUpdateDeleteOps() throws Exception {
         pgHelper.waitTillRecordsAreVerified(0, 5000);
 
         pgHelper.assertRecordCountInPostgres(0);
-
+        // Sleep for 5 minutes to verify connector endpoints
+        System.out.println("Sleeping for 5 minutes");
+        Thread.sleep(300000);
```

Comment:
Is this long a wait required?

Reply:
I saw that.

I added that just to analyze a couple of things with the containers. It's not needed so I have removed it from the PR.